### PR TITLE
Adding poll entity

### LINKED ISSUE #27: Add poll app
Tal depends on the poll to make the event app and then the poll needs a change because it depends on an existing event.
so it will be done by 3 stages.

### PR DESCRIPTION
**Pull request by Sendy459**
Adding poll app
and create Poll model

At the time of creating this PR we still don't have the Event app.

The model has foreign key from Event entity which have not been merged yet, so for now it was not added.

When we add the Event, I'll make the necessary changes in the models.py. resolves #27 and creating the final poll entity.

### CODE REVIEW

**Review by Ofekmn — DISMISSED** [review]

File: fit_meet/settings.py [hunk 1]
```diff
@@ -118,7 +119,7 @@
 
 STATIC_URL = 'static/'
 STATICFILES_DIRS = [
-    BASE_DIR / "static",
+    BASE_DIR / "static_home_page/static",
```
Comment (talko611) [inline fit_meet/settings.py:122]: Already change it in my PR. you don't need to change it as well

File: poll/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
+
+# Register your models here.
```
Comment (talko611) [inline poll/admin.py:3]: You should register your model to the admin app

File: poll/models.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Poll(models.Model):
+    # event = models.OneToOneField(Event)  # waiting for tal to create and merge the event
+    max_suggestions = models.PositiveIntegerField()
+    end_time = models.DateTimeField(default=timezone.now)
```
Comment (talko611) [inline poll/models.py:8]: I don't think giving now as the default end time to the poll when it's been created, is correct. Maybe 2 days from the creation time will be better.

File: poll/models.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.utils import timezone
+from datetime import timedelta
+
```
Comment (Ofekmn) [inline poll/models.py:4]: Hey, I am not sure if you noticed, but the commit didn't pass the flake8 test as there needs to be another empty line here.

**Review by talko611 — APPROVED** [review]

**Review by Ofekmn — APPROVED** [review]

**Review by SaarOrus — DISMISSED** [review]
LGTM

**Comment by kobihk** [comment]
@Sendy459 please update the PR description according to:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
to be able to close issue #27 automatically when we merge this PR

**Comment by kobihk** [comment]
@Sendy459 please rebase

**Review by Ofekmn — DISMISSED** [review]

**Review by talko611 — DISMISSED** [review]

File: poll/models.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+class Poll(models.Model):
+    # event = models.OneToOneField(Event)  # waiting for tal to create and merge the event
```
Comment (ygalblum) [inline poll/models.py:7]: I'm not a fan of committing commented code. I think it's better to remove this line and open a new issue. This way you will remember to add it

**Comment by ygalblum** [comment]
LGTM, but you need to rebase

File: poll/models.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Poll(models.Model):
+    # event = models.OneToOneField(Event)  # waiting for tal to create and merge the event
```
Comment (ormergi) [inline poll/models.py:7]: Please remove this comment, you can comment it on the PR that adds it.

**Review by ormergi — CHANGES_REQUESTED** [review]
Hi Daniel, please squash the second and third commits to the first one.

Fixing review comments should be done on the commit that introduces the change and not in a new commit of its own.
Keeping the commits like this keeps git history clean and easier to follow changes in the code base.

**Review by Ofekmn — DISMISSED** [review]

File: poll/models.py (hunk 6 above)
Comment (ormergi) [reply poll/models.py]: @Sendy459 I see that only part of this is removed, I meant to remove the whole line.
What we have now is dead code (code that will never run), you can suggest adding this to the relevant PR.

**Review by SaarOrus — APPROVED** [review]

**Review by rotem2022 — DISMISSED** [review]

File: poll/models.py (hunk 6 above)
Comment (Sendy459) [reply poll/models.py]: oh ok I thought you meant the second part .

Comment (Sendy459) [reply poll/models.py]: fixed

**Review by ormergi — APPROVED** [review]
lgtm

**Review by ormergi — CHANGES_REQUESTED** [review]
Please change the commit message to describe why this change is necessary and what it does, and add a sign-off.

**Comment by ormergi** [comment]
@Sendy459  the commit message has /n instead of new line.
Regarding the sign-off, you can add the standard sign-off using the `-s` flag: `git commis -s ..` git commit --amend -s ...`

**Review by ormergi — APPROVED** [review]
Thanks for the changes.

**Review by rotem2022 — APPROVED** [review]

**Review by talko611 — APPROVED** [review]

**Pull request merged**